v3/src/utilities/translation/lang/en-US.json5

```
@@ -136,6 +136,7 @@
     "V3.DI.Error.interactiveFrameNotFound": "Interactive Frame not found",
     "V3.DI.Error.itemNotFound": "Item not found",
     "V3.DI.Error.notFound": "Not found",
+    "V3.DI.Error.sharedCaseMetadatanotFound": "SharedCaseMetadata not found",
```

As @tealefristoe mentioned elsewhere, this could be more helpful. How about something like:
```suggestion
    "V3.DI.Error.sharedCaseMetadataNotFound": "Internal error prevented color map access",
```